### PR TITLE
Feature/notaxon

### PR DESCRIPTION
**Pull request by aaronpeikert**

### CODE REVIEW

File: src/taxons/no_taxon.jl [hunk 1]
```diff
@@ -1 +1,38 @@
...
+"""
+struct NoTaxonEver <: NoTaxon end
+NoTaxon() = NoTaxonEver()
+
+
+"""
+    NoTaxonYet(accessdate, modeltype = missing)
+
+A Taxon to show, that there is in fact a model to be coded, but this is at the current point not possible.
+NoTaxonYet requires a timestamp 'accessdate' to further specify at what point there were no possibility to code the respective model.
```
[the remark this answers is not in the document]
Comment (aaronpeikert) [reply src/taxons/no_taxon.jl:23]:
```suggestion
NoTaxonYet requires a timestamp 'accessdate' to further specify at what point there was no possibility to code the respective model.
```

File: src/taxons/no_taxon.jl [hunk 2]
```diff
@@ -1 +1,38 @@
...
+NoTaxonYet gives you the option to name the 'modeltype', you were not able to code, to spare your future self the work of going everything through.
```
[the remark this answers is not in the document]
Comment (aaronpeikert) [reply src/taxons/no_taxon.jl:24]:
```suggestion
NoTaxonYet gives you the option to name the 'modeltype', you were not able to code, to spare your future self the work of going through everything.
```

File: docs/src/tutorials/missing.md [hunk 3]
```diff
@@ -33,10 +33,6 @@ Record
    data: Missing
 ```
 
-```@meta
-DocTestFilters = nothing
-```
-
 As you probably notice we warn you to do that.
 This is to encourage you to think twice, however, after having thought twice about it, you may silence every warning with explicitly suppliyng "empty" instances (except ID, really nothing should hinder you to supply a random id).
```
Comment (valentinkm) [inline docs/src/tutorials/missing.md:37]:
```suggestion
This is to encourage you to think twice. However, after having thought twice about it, you may silence every warning with explicitly suppliyng "empty" instances (except ID, really nothing should hinder you to supply a random id).
```

**Review by valentinkm — CHANGES_REQUESTED** [review]

File: src/abstracttypes.jl [hunk 4]
```diff
@@ -4,12 +4,16 @@ abstract type AbstractMeta end
...
+ - `NoTaxonEver` : A Taxon to show, that there is no SEM to code in this paper
+ - `NoTaxonYet`:  A Taxon to show, that there is in fact a model to be coded, but this is at the current point not possible
+          NoTaxonYet gives you the option to name the 'modeltype', you were not able to code, to spare your future self the work of going everything through.
+          NoTaxonYet is also supposed to give your Record a timestamp 'accessdate' to further specify at what point there were no possibility to code the respective model.
 """
-abstract type NoAbstractTaxon <: Taxon end
+abstract type NoTaxon <: Taxon end
 
 """
 AbstractPathmodel is the supertype of all pathmodels.
```
Comment (LeonieHagitte) [inline src/abstracttypes.jl:19]: """
 `AbstractPathmodel` is the supertype of all pathmodels.

Comment (LeonieHagitte) [inline src/abstracttypes.jl:19]: """
 `AbstractPathmodel` is the supertype of all pathmodels.

**Review by LeonieHagitte — APPROVED** [review]

**Pull request merged**